layouts/shortcodes/embed-pdf.html

```diff
@@ -1,4 +1,5 @@
-<script type="text/javascript" src= '{{"/" | relURL}}/js/pdf-js/build/pdf.js'></script>
+<script src="https://cdn.jsdelivr.net/npm/pdfjs-dist@2.8.335/build/pdf.min.js" integrity="sha256-zRPZmNIE243+AcY4l08d17mLQKv7Srm0y4LLMpV0Ep8=" crossorigin="anonymous"></script>
```

Comment:
I don't know too much about performance, but isn't loading it from CDN going to make it slower?

Also, should you remove the version as to always get the latest.
Also, could we have it fallback to the old implementation of locally copying if this CDN load fails.

Reply:
That is true. In case CDN delivery is slower or blocked there will be some hit.
Let me try and spend some time on this and check how to make it configurable.